### PR TITLE
chore: Project addition by Nishit Baria ✔

### PR DESCRIPTION
**Pull request by Nishitbaria**
## Related Issue

#1624 

## Description

This pull request aims to add the Contentify project to Project Hut,Contentify is a versatile content creation, social media management, and educational resources platform. It provides users with a comprehensive set of tools to excel in content creation, streamline social media management, and access valuable educational resources.

**Changes Made:**
1. Added the Contentify project details to the `DB/projects.json` file, including the project name, description, features, and relevant metadata.
2. Ensured that the Contentify project details adhere to the existing JSON structure and formatting conventions of the `projects.json` file.

### CODE REVIEW

File: DB/projects.json [hunk 1]
```diff
@@ -3925,5 +3931,24 @@
...
+    "YouTube": "#",
+    "Twitter": "https://twitter.com/Nishitbaria1"
+  },
+  "Projects": [
+    {
+      "link": "https://github.com/Nishitbaria/Laughhub",
+      "title": "Laughhub",
+      "description": "Laugh Hub is a hilarious bot with advanced AI voice assistance that offers fun conversations, funny stories, a meme generator, and three to four entertaining games. It's the perfect destination for anyone looking for a good laugh and a break from their daily routine",
+      "tech": ["React JS", "Nodejs"]
+    }
```
Comment (priyankarpal) [inline DB/projects.json:3950]: you've added same projects before

File: DB/projects.json [hunk 2]
```diff
@@ -3814,6 +3814,12 @@
         "title": "Laughhub",
         "description": "Laugh Hub is a hilarious bot with advanced AI voice assistance that offers fun conversations, funny stories, a meme generator, and three to four entertaining games. It's the perfect destination for anyone looking for a good laugh and a break from their daily routine",
         "tech": ["reactjs JS", "nodejs"]
+      } , 
+       {
+        "link": "https://github.com/TechNodes2-0/Contentify",
+        "title": "Contentify",
+        "description": "Contentify is a versatile content creation, social media management, and educational resources platform. It provides users with a comprehensive set of tools to excel in content creation, streamline social media management, and access valuable educational resources.",
+        "tech": ["reactjs JS", "nodejs"]
```
Comment (priyankarpal) [inline DB/projects.json:3822]: add only your projects not others

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: DB/projects.json (hunk 2 above)
Comment (Nishitbaria) [reply DB/projects.json:3822]: Technodes is My Team name where We post our Group Project where me and my Friend are adding those project that we make in hackathon so its not others project :) @priyankarpal

File: DB/projects.json (hunk 1 above)
Comment (priyankarpal) [reply DB/projects.json:3950]: you have to delete duplicate code from here

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by Nishitbaria** [comment]
Hey @priyankarpal  I am Creating New PR for That Because in this Fork there are so many conflict  occuer

**Pull request closed without merging**